sci-libs/blis/blis-0.6.0.ebuild

```diff
@@ -12,19 +12,23 @@ SRC_URI="https://github.com/flame/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc64 ~x86"
-IUSE="openmp pthread serial static-libs eselect-ldso doc 64bit-index"
-REQUIRED_USE="?? ( openmp pthread serial ) ?? ( eselect-ldso 64bit-index )"
+IUSE="doc eselect-ldso openmp pthread serial 64bit-index"
+REQUIRED_USE="
+	?? ( openmp pthread serial )
+	?? ( eselect-ldso 64bit-index )"
 
-RDEPEND="eselect-ldso? ( !app-eselect/eselect-cblas
-						 >=app-eselect/eselect-blas-0.2 )"
+RDEPEND="
+	eselect-ldso? (
+		!app-eselect/eselect-cblas
+		>=app-eselect/eselect-blas-0.2
+	)"
 
 DEPEND="${RDEPEND}
-	${PYTHON_DEPS}
-"
+	${PYTHON_DEPS}"
```

Comment:
`python-any-r1`-`${PYTHON_DEPS}"` in `DEPEND`? :thinking:

Reply:
Done